mysite/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+USER_CHOICES = [
+    ('freelancer', "I'm a Freelancer"),
+    ('recruiter', "I'm a Recruiter"),
+    ]
+
+
+class User(AbstractUser):
+    user_role = models.CharField(max_length=100, choices=USER_CHOICES, default='freelancer')
+
+
+class Project(models.Model):
+    created_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    name = models.CharField(max_length=30)
+    description = models.CharField(max_length=100)
+    status = models.CharField(max_length=30)
+    assigned_to = models.CharField(max_length=30)
+
+
+class ProjectBid(models.Model):
+    bid_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
```

Comment:
please put related_name attribute for all relational fields.

Reply:
Feedback Incorporated.